src/js/components/validateData/ValidateDataFileComponent.jsx

```diff
@@ -312,7 +397,7 @@ export default class ValidateDataFileComponent extends React.Component {
                                     </div>
                                 </div>
                                 {uploadProgress}
-                                <div className="usa-da-validate-item-file-name">
+                                <div className='file-download' onClick={this.clickedReport.bind(this, this.props.item)} download={fileName} rel="noopener noreferrer">
```

Comment:
Font is a different size for file download links when there is a server/header error vs when there are just errors/warnings in the validations